vendor/sdl3/sdl3_render.odin

```diff
@@ -0,0 +1,222 @@
+	SetRenderDrawBlendMode           :: proc(renderer: ^Renderer, blendMode: BlendMode) -> bool ---
+	GetRenderDrawBlendMode           :: proc(renderer: ^Renderer, blendMode: ^BlendMode) -> bool ---
+	RenderClear                      :: proc(renderer: ^Renderer) -> bool ---
+	RenderPoint                      :: proc(renderer: ^Renderer, x, y: f32) -> bool ---
+	RenderPoints                     :: proc(renderer: ^Renderer, points: [^]FPoint, count: c.int) -> bool ---
+	RenderLine                       :: proc(renderer: ^Renderer, x1, y1, x2, y2: f32) -> bool ---
+	RenderLines                      :: proc(renderer: ^Renderer, points: [^]FPoint, count: c.int) -> bool ---
+	RenderRect                       :: proc(renderer: ^Renderer, #by_ptr rect: FRect) -> bool ---
+	RenderRects                      :: proc(renderer: ^Renderer, rects: [^]FRect, count: c.int) -> bool ---
+	RenderFillRect                   :: proc(renderer: ^Renderer, #by_ptr rect: FRect) -> bool ---
+	RenderFillRects                  :: proc(renderer: ^Renderer, rects: [^]FRect, count: c.int) -> bool ---
+	RenderTexture                    :: proc(renderer: ^Renderer, texture: ^Texture, #by_ptr srcrect, dstrect: FRect) -> bool ---
+	RenderTextureRotated             :: proc(renderer: ^Renderer, texture: ^Texture, #by_ptr srcrect, dstrect: FRect, angle: f64, #by_ptr center: FPoint, flip: FlipMode) -> bool ---
+	RenderTextureAffine              :: proc(renderer: ^Renderer, texture: ^Texture, #by_ptr srcrect: FRect, #by_ptr origin: FPoint, #by_ptr right: FPoint, #by_ptr down: FPoint) -> bool ---
+	RenderTextureTiled               :: proc(renderer: ^Renderer, texture: ^Texture, #by_ptr srcrect: FRect, scale: f32, #by_ptr dstrect: FRect) -> bool ---
+	RenderTexture9Grid               :: proc(renderer: ^Renderer, texture: ^Texture, #by_ptr srcrect: FRect, left_width, right_width, top_height, bottom_height: f32, scale: f32, #by_ptr dstrect: FRect) -> bool ---
+	RenderGeometry                   :: proc(renderer: ^Renderer, texture: ^Texture, vertices: [^]Vertex, num_vertices: c.int, indices: [^]c.int, num_indices: c.int) -> bool ---
+	RenderGeometryRaw                :: proc(renderer: ^Renderer, texture: ^Texture, xy: [^]f32, xy_stride: c.int, color: [^]FColor, color_stride: c.int, uv: [^]f32, uv_stride: c.int, num_vertices: c.int, indices: rawptr, num_indices: c.int, size_indices: c.int) -> bool ---
+	RenderReadPixels                 :: proc(renderer: ^Renderer, #by_ptr rect: Rect) -> ^Surface ---
+	RenderPresent                    :: proc(renderer: ^Renderer) -> bool ---
+	DestroyTexture                   :: proc(texture: ^Texture) ---
+	DestroyRenderer                  :: proc(renderer: ^Renderer) ---
+	FlushRenderer                    :: proc(renderer: ^Renderer) -> bool ---
+	GetRenderMetalLayer              :: proc(renderer: ^Renderer) -> rawptr ---
+	GetRenderMetalCommandEncoder     :: proc(renderer: ^Renderer) -> rawptr ---
```

Comment:
If you found it a good idea in sdl3_metal.odin to use the darwin types, they can also be applied here

Reply:
So I was just copying the headers exactly. And they still don't use the Darwin types directly either because of compilation issues.